README.md

```diff
@@ -169,6 +169,18 @@ vi /etc/kubernetes/manifests/kube-apiserver.yaml
 - --insecure-bind-address=0.0.0.0
 ```
 
+KubeEdge also supports https connection to Kubernetes apiserver. Follow the steps in [Kubernetes Documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to create the kubeconfig file.
+
+Enter the path to kubeconfig file in controller.yaml
+```yaml
+controller:
+  kube:
+    .
```

Comment:
what about `...` instead of repeating 3 lines of `.`?

Reply:
Done, Thanks